projects/1.20/assets/create-connected/create-connected/lang/zh_cn.json

```
@@ -0,0 +1,244 @@
+    "block.create_connected.linked_oak_button": "无线橡木按钮",
+    "block.create_connected.linked_polished_blackstone_button": "无线磨制黑石按钮",
+    "block.create_connected.linked_spruce_button": "无线云杉木按钮",
+    "block.create_connected.linked_stone_button": "无线石头按钮",
+    "block.create_connected.linked_warped_button": "无线诡异木按钮",
+    "block.create_connected.overstress_clutch": "过应力离合器",
+    "block.create_connected.overstress_clutch.tooltip.behaviour1": "_重置_至耦合状态。",
+    "block.create_connected.overstress_clutch.tooltip.condition1": "手持扳手右击时",
+    "block.create_connected.overstress_clutch.tooltip.summary": "当网络应力_过载_时，离合器会在设定的延迟后_断开_。",
+    "block.create_connected.parallel_gearbox": "平行齿轮箱",
+    "block.create_connected.sequenced_pulse_generator": "可编程脉冲生成器",
+    "block.create_connected.shear_pin": "剪切销",
+    "block.create_connected.shear_pin.tooltip.summary": "一次性连接器，当动力网络_过载_时会_断裂_。",
+    "block.create_connected.six_way_gearbox": "六向齿轮箱",
+    "block.create_connected.six_way_gearbox.tooltip.summary": "_六个面_都有轴的齿轮箱。顶部和底部以_半速_旋转。",
+    "block.create_connected.wrapped_copycat_fence": "包装伪装栅栏",
+    "block.create_connected.wrapped_copycat_fence_gate": "包装伪装栅栏门",
+    "block.create_connected.wrapped_copycat_stairs": "包装伪装楼梯",
+    "block.create_connected.wrapped_copycat_wall": "包装伪装墙",
+    "create.gui.sequenced_gearshift.instruction.loop": "循环",
+    "create.gui.sequenced_gearshift.instruction.loop.descriptive": "重新启动整个序列",
+    "create.gui.sequenced_gearshift.instruction.turn_await": "维持",
+    "create.gui.sequenced_gearshift.instruction.turn_await.descriptive": "旋转直到新的红石脉冲",
+    "create.gui.sequenced_gearshift.instruction.turn_time": "时间",
+    "create.gui.sequenced_gearshift.instruction.turn_time.descriptive": "旋转指定时间",
+    "create.gui.sequenced_gearshift.instruction.turn_time.duration": "持续时间",
+    "create.item_attributes.id_contains": "在ID中有词语 \"%1$s\"",
+    "create.item_attributes.id_contains.inverted": "在ID中没有词语 \"%1$s\"",
+    "create.item_attributes.max_damage": "有 %1$s 最大耐久度",
+    "create.item_attributes.max_damage.inverted": "没有 %1$s 最大耐久度",
+    "create.item_attributes.stack_size": "最大堆叠数为 %1$s",
+    "create.item_attributes.stack_size.inverted": "最大堆叠数不为 %1$s",
+    "create_connected.centrifugal_clutch.speed_threshold": "速度阈值",
+    "create_connected.gui.overstress_clutch.uncoupled": "未耦合",
+    "create_connected.gui.overstress_clutch.uncoupled_explanation": "该离合器因应力过载_断开_。请修复网络，然后用_扳手_或_红石脉冲信号__重置_该离合器。",
+    "create_connected.gui.sequenced_pulse_generator.instruction": "指令",
+    "create_connected.gui.sequenced_pulse_generator.instruction.end": "结束",
+    "create_connected.gui.sequenced_pulse_generator.instruction.end.descriptive": "结束",
+    "create_connected.gui.sequenced_pulse_generator.instruction.loop": "循环",
+    "create_connected.gui.sequenced_pulse_generator.instruction.loop.descriptive": "永远循环",
+    "create_connected.gui.sequenced_pulse_generator.instruction.loop_for": "循环",
```

原文不一致；如无特殊要求，应当在译文中也区分。
```suggestion
    "create_connected.gui.sequenced_pulse_generator.instruction.loop_for": "定次循环",
```